charts/openyurt/templates/pool-coordinator.yaml

```diff
@@ -0,0 +1,209 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: pool-coordinator-apiserver
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    openyurt.io/topologyKeys: openyurt.io/nodepool
+  labels:
+    name: pool-coordinator
+spec:
+  type: ClusterIP
+  ports:
+    - port: 443
+      targetPort: {{ .Values.poolCoordinator.apiserverPort }}
+      protocol: TCP
+      name: https
+  selector:
+    k8s-app: pool-coordinator
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pool-coordinator-etcd
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    openyurt.io/topologyKeys: openyurt.io/nodepool
+  labels:
+    name: pool-coordinator
+spec:
+  type: ClusterIP
+  ports:
+    - port: 2379
+      targetPort: {{ .Values.poolCoordinator.etcdPort }}
+      protocol: TCP
+      name: https
+  selector:
+    k8s-app: pool-coordinator
+---
+apiVersion: apps.openyurt.io/v1alpha1
+kind: YurtAppDaemon
+metadata:
+  name: pool-coordinator
```

Comment:
maybe we need to add `nodeSelector` to make `pool-coordinator` pod can work on edge nodes only.
```
openyurt.io/is-edge-worker: true
```

Reply:
Maybe a node pool selector？Or it will failed on the node pool which is not an edge node pool.
But I didn't see a node pool selector for node pool type. There is just selector for node pool label.

Reply:
yes, we need to skip cloud node pool, so we need to make sure that pool-coordinator will not be scheduled into it.

Reply:
As we discussed in IM, we add pool-coordinator to all node pool now which includes cloud type node pool.
When yurtappdaemon supported to select node pool by type, we should update pool-coordinator chart again.